development/commit-messages.md

```diff
@@ -0,0 +1,29 @@
+# Commit messages
+
+Write the summary line and description of what you have done in the imperative
+mode, that is as if you were commanding someone. For example, start the line
+with "Fix", "Add" or "Change" instead of "Fixed", "Added" or "Changed".
+
+Limit your commit summary (the first line of the commit) to 50 characters. If
+you need to provide more detail (which you often will), leave a blank line and
+add a commit description.
+
+You should make sure you capture the context and reasoning for the change:
+
+- Why is the change necessary?
+- How does the commit address the issue?
+- What are the consequences of the change?
+
+You should link to any supporting information:
+
+- relevant guidance, third party documentation or blog posts that informed
+  your decision
+- Trello cards, GitHub issues or other bug reports
+
```

Comment:
Could we add: "Trello cards/bug reports might not be viewable by everyone so don't rely on them exclusively to capture the information about your changes."